File: Selenium/Activity10.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver import ActionChains
with webdriver.Chrome() as driver:
    action = ActionChains(driver)
    driver.get("https://training-support.net/webelements/drag-drop")
    print("Web age title - " + driver.title)
    ele1 = driver.find_element(By.ID, "ball")
    ele2 = driver.find_element(By.ID, "dropzone1")
    ele3 = driver.find_element(By.ID, "dropzone2")
    action.click_and_hold(ele1).move_to_element(ele2).perform()
    if(driver.find_element(By.CLASS_NAME, "dropzone-text").text == "Dropped!"):
        print("Element moved to Zone1")
    action.click_and_hold(ele1).drag_and_drop(ele2,ele3).perform()
    if(driver.find_element(By.CLASS_NAME, "dropzone-text").text == "Dropped!"):
        print("Element moved to Zone2")

File: Selenium/Activity10.java
package Selenium;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity10 {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		Actions action = new Actions(driver);
		driver.manage().window().maximize();
		driver.get("https://training-support.net/webelements/drag-drop");
		System.out.println("Page Title is : " + driver.getTitle());
		WebElement ele1 = driver.findElement(By.id("ball"));
		WebElement ele2 = driver.findElement(By.xpath("//div[@id='dropzone1']"));
		WebElement ele3 = driver.findElement(By.xpath("//div[@id='dropzone2']"));
		action.clickAndHold(ele1).moveToElement(ele2).build().perform();
		if(ele2.findElement(By.className("dropzone-text")).getText().equals("Dropped!")) {
        	System.out.println("Ball was dropped in Dropzone 1");
        }
		action.clickAndHold(ele1).dragAndDrop(ele2,ele3).build().perform();
		if(ele3.findElement(By.className("dropzone-text")).getText().equals("Dropped!")) {
        	System.out.println("Ball was dropped in Dropzone 2");
        }
		driver.quit();
	}

}
